src/meta/meta_bulk_load_service.cpp

```diff
@@ -1713,10 +1756,7 @@ void bulk_load_service::do_continue_app_bulk_load(
         gpid pid = gpid(app_id, i);
         partition_bulk_load(ainfo.app_name, pid);
         if (app_status == bulk_load_status::BLS_INGESTING) {
-            tasking::enqueue(
-                LPC_BULK_LOAD_INGESTION,
-                _meta_svc->tracker(),
-                std::bind(&bulk_load_service::partition_ingestion, this, ainfo.app_name, pid));
+            partition_ingestion(ainfo.app_name, pid);
```

Comment:
Will this method be triggred periodically？ like `node balalance check`? if only execute after `update status` or `last response replay`, when one send failed, can we make sure all the replica can receive `ingest request`?